meinberlin/apps/budgeting/assets/FilterBar.jsx

```diff
@@ -37,43 +37,32 @@ export const FilterBar = props => {
   }
 
   const prepareFilter = (filter, filterType) => {
```

Comment:
> :last-child {
            float: right;
            margin-right: 0;
        }

Reply:
I guess a question to @khamui ?